lms/static/scripts/frontend_apps/components/GroupConfigSelector.tsx

```diff
@@ -45,8 +53,8 @@ function GroupSelect({ busy, groupSets, onInput, selectedGroupSetId }) {
       <select
         disabled={busy}
         id={selectId}
-        onInput={e =>
-          onInput(/** @type {HTMLInputElement} */ (e.target).value || null)
+        onInput={(e: Event) =>
+          onInput((e.target as HTMLSelectElement | null)?.value || null)
```

Comment:
I would like to have a better alternative here, but according to preact docs, there's no generic event type and you need to check the type yourself.

Since we were already doing an on-the-fly type casting, I've left it as it was.